capstone/src/main/webapp/index.html

```diff
@@ -53,10 +53,11 @@ <h3 class="mdc-drawer__title">EipCURE</h3>
           </section>
           <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end">
             <label class="search-label top-app-bar-text">Search</label>
-            <form action="/" method="get" id="search-box">
+            <form action="/restaurants" method="post" id="search-box">
```

Comment:
How come we're switching to a post, for a basic search box request?

Reply:
I changed it to a GET request, and also changed the request path to `/search` because I think it makes a bit more sense.